packages/frontend/src/pages/Executions/index.tsx

```diff
@@ -49,7 +49,13 @@ export default function Executions(): React.ReactElement {
   return (
     <Box sx={{ py: 3 }}>
       <Container variant="page">
-        <Grid container sx={{ mb: [0, 3] }} columnSpacing={1.5} rowSpacing={3}>
+        <Grid
+          container
+          sx={{ mb: [0, 3] }}
```

Comment:
`sx` makes me itchy but out of scope for this PR

Reply:
i changed this to use flex anw fyi